jupyter-extension/src/plugins/mount/components/Pipeline/hooks/usePipeline.tsx

```diff
@@ -101,7 +108,10 @@ export const usePipeline = (
         }
       } catch (e) {
         if (e instanceof ServerConnection.ResponseError) {
-          setErrorMessage('Error creating pipeline');
+          // statusText is the only place that the user will get yaml parsing
+          // errors (though it will also include Pachyderm errors, e.g.
+          // "missing input repo").
+          setErrorMessage('Error creating pipeline: ' + e.response.statusText);
```

Comment:
We've gone back and forth a couple times on this code. I believe @bbonenfant looked into this more and couldn't get the error returned from the server?

Reply:
Yeah I'm a little torn--if we're doing the YAML parsing in python, it seems important that those errors get shown, so I wanted to display *something*. I got the code here from playing around in the browser console, as far as that goes. This is what it looks like:
<img src="https://user-images.githubusercontent.com/235674/234050597-a73987fa-44de-4904-8720-6dc480187f50.png" width=200>

Reply:
There's also a funny UX sort-of-bug that I'm looking into now (which you'll likely find if you start playing with the error reporting), which is that `yaml.safe_load()` seems much more flexible than I expected, and accepts stuff that I actually wish it would reject. I'm trying to figure out if I can parse the YAML into an input spec rather than a generic dict for error reporting purposes.

That issue doesn't seem catastrophic to me because `python_pachyderm` will still reject those specs when we try to turn the dict into a `CreatePipelineRequest`, but the error message is a little opaque

Reply:
Note for posterity: our Jira ticket about surfacing the right error message is [INT-934]